### ISSUE TITLE
[Feature]: Can you add Async Select Field ?

### ISSUE THREAD

**Issue by Uhasith** [luvi-ui/laravel-luvi#4]
### What happened?

From that, we can define a route and get select options from an endpoint. Basically user can type and Search the record. This is useful when dealing with Large sets of data.

### How to reproduce the bug

From that, we can define a route and get select options from an endpoint. Basically user can type and Search the record. This is useful when dealing with Large sets of data.

### Package Version

2.0.0

### PHP Version

8.2

### Laravel Version

11.0.0

### Which operating systems does this happen with?

macOS

**Comment by Uhasith** [comment]
> What you're describing is the [shadcn/ui combobox](https://ui.shadcn.com/docs/components/combobox). We elaborate which solution we will offer right now. We use luvi-ui in a livewire project at the moment. We will add a showcase how to solve that problem with livewire.

Yeah, correct. In wireui they have this kind of thing [https://wireui.dev/components/select#async-search](url). You can define a Laravel get route and define a logic there as you want. This is handy thing to have.

**Comment by Uhasith** [comment]
When can we have the Combo Box Feature ?  @thoresuenert

**Comment by thoresuenert** [comment]
@Uhasith we have an example ready, need to polish some things tomorrow. hopefully you will release it tomorrow.

**Comment by thoresuenert** [comment]
@Uhasith we cooked something for you https://laravel-luvi.com/docs/livewire-components/combobox hopefully that is the answer to your question. 

Otherwise feel free to elaborate more about your needs or implement it yourself. Happy to see pull requests.

**Comment by Uhasith** [comment]
> @Uhasith we cooked something for you https://laravel-luvi.com/docs/livewire-components/combobox hopefully that is the answer to your question.
> 
> Otherwise feel free to elaborate more about your needs or implement it yourself. Happy to see pull requests.

Yeah it's looks awesome. But i guess it should be awesome to declare some backend route for handle the search query. From that kind of approach we will have more flexibility to perform the Search. For example when we dealing with Large amount of data set. We can set Limits. Pass some parameters and Narrow down our search Async Way. For update forms we can pass selected ids so they will come at the Top when search route is called. Really Really Happy about the effort Guys. But this can be optimized. Refer Some Docs in wireui Documentation. ❤️🚀🚀 @thoresuenert

**Comment by Uhasith** [comment]
> > What you're describing is the [shadcn/ui combobox](https://ui.shadcn.com/docs/components/combobox). We elaborate which solution we will offer right now. We use luvi-ui in a livewire project at the moment. We will add a showcase how to solve that problem with livewire.
> 
> Yeah, correct. In wireui they have this kind of thing https://wireui.dev/components/select#async-search. You can define a Laravel get route and define a logic there as you want. This is handy thing to have.

@thoresuenert  Check these Mate. You can get a clear idea. Only thing i miss in Luvi Ui at the moment is this kind of Fully featured ComboBox 😃

**Comment by thoresuenert** [comment]
We are open to PRs. Feel free to do it as you need.

**Comment by 18764-MM1079-MUHAMMADMAAZ** [comment]
**Can you add the Layouts (Header & Sidebar) to this please as in the Flux Ui?**

_Your current project is incomplete without these components._
